playbooks/common/openshift-cluster/upgrades/post_control_plane.yml

```diff
@@ -51,6 +51,9 @@
       '{"spec":{"template":{"spec":{"containers":[{"name":"router","image":"{{ router_image }}","livenessProbe":{"tcpSocket":null,"httpGet":{"path": "/healthz", "port": 1936, "host": "localhost", "scheme": "HTTP"},"initialDelaySeconds":10,"timeoutSeconds":1}}]}}}}'
       --api-version=v1
     with_items: "{{ haproxy_routers }}"
+    # AUDIT:changed_when_note: `false` not being set here. What we
```

Comment:
@sdodson sounds good to me. We should do that when we begin doing the actual change sweep, IMHO. Right now the grep I use for "we're cool" is 

```
grep -r 'AUDIT:changed_when:' roles playbooks
```

And for ones which I just added thoughts around

```
grep -r 'AUDIT:changed_when_' roles playbooks
```